reVX/least_cost_xmission/aoswt_utilities.py

```diff
@@ -100,15 +100,15 @@ class CombineRasters:
     LAND_MASK_FNAME = 'land_mask.tif'
 
     # Slope cutoffs (degrees)
-    slope_barrier_cutoff = 15  # slopes >= this value are for barriers & high
+    SLOPE_BARRIER_CUTOFF = 15  # slopes >= this value are for barriers & high
```

Comment:
These are not supposed to be constant, but attributes that can be changed by the consumer code.

Reply:
I guess not constant in the literal sense but more like representing a number. If these are going to be changed frequently, they should probably be moved to the `__init__`. Otherwise they are "constant enough" to warrant uppercase :p